packages/flex-plugin-scripts/src/scripts/start.ts

```diff
@@ -165,7 +199,27 @@ export const _updatePluginsUrl = (port: number) => {
 
   // Replace port and re-write to file
   plugins[pluginIndex].src = plugins[pluginIndex].src.replace(matches[1], port.toString());
-  fs.writeFileSync(paths().app.pluginsJsonPath, JSON.stringify(plugins, null, 2));
+  fs.writeFileSync(getPaths().app.pluginsJsonPath, JSON.stringify(plugins, null, 2));
+};
+
+/**
+ * Update port of a plugin
+ * @param port
+ * @param name
+ */
+export const _updatePluginPort = (port: number, name: string) => {
+  const config = readPluginsJson();
+  config.plugins = config
```

Comment:
You actually don't need a map because in JavaScript objects are passed by reference - so you can do:

```
config.plugins.forEach((plugin) => {
  if (plugin.name === name) {
    plugin.port = port;
  }
});
```